emily/handler/tests/integration/accessors.rs

```diff
@@ -0,0 +1,57 @@
+//! Integration tests for the database accessors
+
+use aws_sdk_dynamodb::operation::update_item::builders::UpdateItemFluentBuilder;
```

Comment:
There are a few things in this file that appear to be unused.

Reply:
It's my continuous transgression of leaving in things I intended to use in the next PR, but you're right its definitely better that I delete them.